src/Autofac.Integration.Wcf/Autofac.Integration.Wcf.nuspec

```diff
@@ -4,7 +4,7 @@
     <id>Autofac.Wcf</id>
     <version>$version$</version>
     <authors>Autofac Contributors</authors>
-    <licenseUrl>https://opensource.org/licenses/mit-license.php</licenseUrl>
+    <license type="file">LICENSE</license>
```

Comment:
Use `<license type="expression">MIT</license>` - this is [how we do it in core Autofac, too](https://github.com/autofac/Autofac/blob/83ab65cf754507716d4d577e171072c1eab05625/src/Autofac/Autofac.csproj#L21)